source/ed/connectors/gtfs_parser.cpp

```diff
@@ -861,7 +861,12 @@ void TripsGtfsHandler::handle_line(Data& data, const csv_row& row, bool) {
 
     nm::Line* line = it->second;
 
-    nm::Route* route = get_or_create_route(data, {line, row[direction_id_c]});
+    // direction_id is optional (and possible values "0" or "1"), so defaulting to "0"
+    std::string direction_id = "0";
```

Comment:
gtfs spec says that it should be either `0` or `1`, but since it is a string, we decided to be a bit more laxist and accept any direction.

With it, even with GTFS, you can create any number of routes for a given line (as we can do with ntfs)